controllers/storageclassrequest/storageclassrequest_controller.go

```diff
@@ -389,6 +411,7 @@ func (r *StorageClassRequestReconciler) reconcileCephFilesystemSubVolumeGroup(st
 		}
 
 		addLabel(r.cephFilesystemSubVolumeGroup, controllers.StorageConsumerNameLabel, r.storageConsumer.Name)
+		addLabel(r.cephFilesystemSubVolumeGroup, controllers.StorageProfileSpecLabel, storageProfile.GetSpecHash())
```

Comment:
why we are adding this to SVG as well?

Reply:
Good question. In the case of SVG we do want a separate  SVG for each SCC (at least that is my opinion). In that case, adding the label does not add any additional value